socialhome/streams/templatetags/json_context.py

```diff
@@ -0,0 +1,39 @@
+from django.utils.safestring import mark_safe
+from django.template import Library
+import json
+
+register = Library()
+
+
+@register.simple_tag(takes_context=True)
+def json_context(context):
+    """Generate a JS <script> tag from context
+
+    Serializes ``context["json_context"]`` into JSON and generates a JS
+    script to attach it to ``window.context``
+
+    :param context: Current view context
+    :return: ``<script>window.context = {<context["json_context"]>}</script>``
+    """
+    return _json_context(context, lambda obj: str(obj))
+
+
+@register.simple_tag(takes_context=True)
+def json_context_unsafe(context):
```

Comment:
I really would like this to be removed if it's only for debugging purposes. It is introducing code that is never executed into the project. If you really want, you could add a `raise_error=False` parameter to `json_context` function and use that to decide whether to raise. This is quite a common way to control functions that can either silently "do a bit more" or be strict and fail early. As long as you then write a test with and without the raise parameter.